src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
```

Comment:
CSS: Avoid using the `body` tag for styling. It is better to use class selectors. The only exceptions are the `html` and `body` tags, but even then, it's better to use classes whenever possible.